CONTRIBUTING.md

```diff
@@ -195,6 +196,46 @@ copyrights.
 
 You can include your individual copyright after any existing copyrights.
 
+## **GFXReconstruct PR Process**
+
+### Creating development branches
+
+Create your branch in your personal fork of [LunarG/gfxreconstruct](https://github.com/LunarG/gfxreconstruct).  We recommend using a name describing the change, e.g. `fix-XYZ` (where `XYZ` is the issue number) or `name-of-feature`, based on the `dev` branch.
+> E.g.  `fix-341` or `track-parent-info`
+
+### During Development
+
+* Don’t hand-edit C++ headers or implementation files in `framework/generated`.  To change those files, edit the Python generator scripts and run the generator (`python3 generate_vulkan.py` or `python3 generate_dx12.py`).
+* Review [earlier sections](#coding-conventions-and-formatting) of this document for coding style guidelines.
+* Attempt to follow the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html). 
+* Attempt to follow the [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines). 
+* Use `GFXRECON_ASSERT `and not `assert.`
+* Prefer explicit comparison against `nullptr.`
+    * e.g. use `if(pointer_variable == nullptr).`
+* Avoid editing code unrelated to the new functionality or bugfix. We want to be able to cleanly revert your commits later if there turns out to be a problem with them, and keeping PRs atomic helps with that.
```

Comment:
I added the explanation of at least one reason for not cluttering PRs with unrelated changes.